src/emailer/agent.py

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8 -*- {{{
+# ===----------------------------------------------------------------------===
+#
+#                 Installable Component of Eclipse VOLTTRON
+#
+# ===----------------------------------------------------------------------===
+#
+# Copyright 2022 Battelle Memorial Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# ===----------------------------------------------------------------------===
+# }}}
+
+
+
+from collections import defaultdict
+
+# Import the email modules we'll need
+from email.mime.text import MIMEText
+import logging
+import socket
+
+# Import smtplib for the actual sending function
+import smtplib
+import sys
+
+import gevent
+from volttron.utils import get_utc_seconds_from_epoch
+from volttron.client.vip.agent import Agent, Core, PubSub
+from volttron import utils
+from volttron.client.messaging import topics
+from volttron.client.messaging.health import ALERT_KEY, STATUS_BAD, Status, \
+    STATUS_GOOD
+from volttron.client.vip.agent import Agent
+
+utils.setup_logging()
+_log = logging.getLogger(__name__)
+__version__ = '1.4.2'
```

Comment:
This version should not be different than the version installed...we need to sync these better...

Reply:
with the version installed. What version of what? the volttron version or the agent watcher volttron? what is this for?

Reply:
this is the version of this particular agent
i.e. the pip version